src/scales/quantitativeScale.ts

```diff
@@ -256,6 +257,24 @@ export class QuantitativeScale<D> extends Scale<D, number> {
     }
   }
 
+  public magnify(magnifyAmount: number, centerValue: number) {
```

Comment:
Would it make sense for the `Plottable.Scale` class to declare abstract methods for `magnify`/`translate`/`scaleTransformation`/`getTransformationDomain` ? And then make quantitativeScale abstract, to pass down scaleTransformation and getTransformationDomain?